setup.py

```diff
@@ -51,8 +40,6 @@ def alltests():
         include_package_data=True,
         install_requires=["ZConfig", "setuptools"],
         extras_require=dict(test=tests_require),
-        test_suite='__main__.alltests',
-        tests_require=tests_require
     )
 except ImportError:
     from distutils.core import setup
```

Comment:
Is this still needed? Do we need to support installing `zdaemon` without having setuptools installed?

Reply:
I'd say no.

Reply:
I dropped it in https://github.com/zopefoundation/zdaemon/pull/26/commits/bf1875b7768dda75e37735768dc0d2ef60934ce4. Could you please take another look at my changes?